iaso/cli/__init__.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+load_dotenv(dotenv_path=(Path(".") / ".env"))
+
+import click_completion
+import click
+import click_completion.core
+
+
+def completion_startswith(string, incomplete):
+    """A custom completion matching that supports case insensitive matching"""
+    if os.environ.get("_CMD_IASO_CASE_INSENSITIVE_COMPLETE"):
+        string = string.lower()
+        incomplete = incomplete.lower()
+    return string.startswith(incomplete)
+
+
+click_completion.core.startswith = completion_startswith
+click_completion.init()
+
+
+from ..click.lazy import LazyCommandGroup
```

Comment:
Although this way of importing things is common in python notebooks, I don't think it is PEP8 compliant, and it should be reviewed through the code.
All imports are always at the top of the source code file.

Reply:
This has been fixed in 16ceb2088ac15c52f8e0d3ccefeeb70b35a81645